Meadowlark-js/tests/system/crud/Upsert.test.ts

```diff
@@ -0,0 +1,224 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { FrontendRequest, FrontendResponse, get, newFrontendRequest, upsert, update } from '@edfi/meadowlark-core';
+import { getNewClient, getCollection, resetSharedClient } from '@edfi/meadowlark-mongodb-backend';
+import { MongoClient } from 'mongodb';
+
+jest.setTimeout(40000);
+process.env.DOCUMENT_STORE_PLUGIN = '@edfi/meadowlark-mongodb-backend';
+process.env.SIGNING_KEY =
+  'v/AbsYGRvIfCf1bxufA6+Ras5NR+kIroLUg5RKYMjmqvNa1fVanmPBXKFH+MD1TPHpSgna0g+6oRnmRGUme6vJ7x91OA7Lp1hWzr6NnpdLYA9BmDHWjkRFvlx9bVmP+GTave2E4RAYa5b/qlvXOVnwaqEWzHxefqzkd1F1mQ6dVNFWYdiOmgw8ofQ87Xi1W0DkToRNS/Roc4rxby/BZwHUj7Y4tYdMpkWDMrZK6Vwat1KuPyiqsaBQYa9Xd0pxKqUOrAp8a+BFwiPfxf4nyVdOSAd77A/wuKIJaERNY5xJXUHwNgEOMf+Lg4032u4PnsnH7aJb2F4z8AhHldM6w5jw==';
+
+const headers = {
+  Authorization:
+    'bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJlZC1maS1tZWFkb3dsYXJrIiwiYXVkIjoibWVhZG93bGFyayIsInN1YiI6InN1cGVyLWdyZWF0LVNJUyIsImp0aSI6IjNkNTliNzVmLWE3NjItNGJhYS05MTE2LTE5YzgyZmRmOGRlMyIsImlhdCI6MTYzNjU2MjA2MCwiZXhwIjozODQ1NTQ4ODgxfQ.WF5ABFZAvNsLDZktwa8VxDR846fGptRXJObtQitbL8I',
+};
```

Comment:
Eventually I imagine we'll have a plugin system for authentication, and we could inject our current code as a "TestHarnessOAuthPlugin" or something like that.